.vscode/settings.json

```diff
@@ -31,5 +31,6 @@
     "source.organizeImports": true
   },
   "eslint.format.enable": true,
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "C_Cpp.default.configurationProvider": "ms-vscode.cmake-tools"
```

Comment:
Please don't change this setting on us :)

Reply:
I correct it

thanks :)